tests/php/Dev/DeprecationTest.php

```diff
@@ -111,7 +111,7 @@ public function testNoticeNoReplacement()
             'Will be removed without equivalent functionality to replace it.',
             'Called from SilverStripe\Dev\Tests\DeprecationTest->testNoticeNoReplacement.'
         ]);
-        $this->expectDeprecation();
+        $this->expectException(DeprecationTestException::class);
```

Comment:
This fixes `Error: Call to undefined method SilverStripe\Dev\Tests\DeprecationTest::expectDeprecation()` which happened after recently merging up https://github.com/silverstripe/silverstripe-framework/pull/11429